desktop-environment.el

```diff
@@ -176,6 +176,16 @@ portion of the screen."
   "Shell command toggling bluetooth."
   :type 'string)
 
+
+;;; Customization - EXWM keybindings
+
+(defcustom desktop-environment-update-exwm-global-keys :global
```

Comment:
why `:global` and not `'global`? I never understood the real difference. If you have some pointer to the documentation, I will happily have a look.

Reply:
For me, at least, it's a habit which mainly stems from Common Lisp.  The differences regarding namespace issues don't apply to elisp, but in general they are used wherever you would choose between a set of predefined options.

From https://www.gnu.org/software/emacs/manual/html_node/elisp/Symbol-Type.html:

> A symbol whose name starts with a colon (‘:’) is called a keyword symbol. These symbols automatically act as constants, and are normally used only by comparing an unknown symbol with a few specific alternatives.